Analyzer/plugins/Analyzer.cc

```diff
@@ -1533,7 +1599,7 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
                                                 CutTOF_Flip_);
     } else {
       // Preselection not passed, skipping it
-      continue;
+      //continue; // NOOOOOO !! DONT SKIP IT ! SKIP CAUSES PROBLEM FOR THE TUPLE
```

Comment:
well but I need to skip it for the histo based code :D not sure what we should do...

Reply:
Really? I'm not sure because you fill the histograms in the PassPreselection function

Reply:
Oh ok it's for the ones passing preselection...

Reply:
I fixed this in https://github.com/CMS-HSCP/SUSYBSMAnalysis-HSCP/pull/43